samples/peripheral/802154_phy_test/Kconfig

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "PHY Test Tool configuration"
+
+choice
+	prompt "Device Mode"
+	default BOTH
+	help
+	  Select the device mode
+
+config DUT_MODE
+	bool "Configure device as a DUT"
```

Comment:
These options should be extended with help string describing what is DUT and what is CMD. These names are ambiguous and misleading.